.travis.yml

```diff
@@ -12,11 +12,15 @@ matrix:
   - python: 2.7
     env: TEST_TARGET=cli
   - python: 2.7
+    env: TEST_TARGET=build_cbi
```

Comment:
We should run tests that require an external connection in the `allow_failures` section so we can debug them quickly.

Note to self. The current setup is not smart, it runs a lot of test in both scenarios. I'll figure out what depends on `build_cbi` in another PR and optimize this.